plugins/action/action.h

```diff
@@ -46,7 +45,8 @@ class Action : public PluginBase
      * @brief Possible results returned for commanded actions.
      */
     enum class Result {
-        SUCCESS = 0, /**< @brief Success. The action command was accepted by the vehicle. */
+        UNKNOWN, /**< @brief Unspecified error. */
```

Comment:
I put `UNKNOWN` at the top of the list, so that we can use the easy `static_cast<>` when converting to the gRPC enum. Having `UNKNOWN` at the end of the list feels wrong because it will need to keep its index (meaning that if we add a new value, UNKNOWN will not be the last one anymore) or we will need more boilerplate for the conversion in `ActionServiceImpl`, which is not desirable.

Reply:
Oh, I actually ran into this before. However, it made sense because a new error code added is essentially unknown. So then I would always just shift `UNKNOWN` down but I guess that's not an option in proto?

Reply:
No. In proto, `UNKNOWN = 0` will stay forever. If you remove `UNKNOWN`, you cannot use "0" for any other value anymore, forever :-). So we cannot count on an easy conversion from the C++ enum to the proto one, which is why I am very careful with the `static_cast<>` we currently do (hence the unit tests).